vehicle/raw_vehicle_cmd_converter/src/csv_loader.cpp

```diff
@@ -143,8 +143,6 @@ double CSVLoader::clampValue(
   const double max_value = *std::max_element(ranges.begin(), ranges.end());
   const double min_value = *std::min_element(ranges.begin(), ranges.end());
   if (val < min_value || max_value < val) {
-    std::cerr << "Input " << name << ": " << val << " is out of range. use closest value."
-              << std::endl;
```

Comment:
Should we keep this? I don't know why this is removed.

Reply:
this prints every loop like while stopped so too noisy.

Reply:
That means, the map is not well defined. Then, we should change the message to ` value is out of range. Use the closest value. Please update the conversion map` or something.

Reply:
understood that's better!!
[update error message](https://github.com/autowarefoundation/autoware.universe/pull/5688/commits/49277348bbe37d05cc8dde4d87910c678be5f146)